google/resource_compute_instance.go

```diff
@@ -563,6 +564,13 @@ func resourceComputeInstance() *schema.Resource {
 				},
 			},
 
+			"status": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "RUNNING",
```

Comment:
```suggestion
```

Users who've stopped their instance out of band would see a diff with a default. Combined with the changes to `desired_state` that make state management optional below, I don't think we need this.